packages/strapi-plugin-content-type-builder/admin/src/translations/sk.json

```diff
@@ -0,0 +1,154 @@
+{
+  "attribute.boolean": "Boolean",
```

Comment:
Do you mind removing the "not translated keys" there's a fallback to English. This way we can automatically create issues on GitHub asking people from the community to fill the missing translations?

Thanks a lot